include/xsimd/arch/xsimd_avx512f.hpp

```diff
@@ -1317,7 +1317,7 @@ namespace xsimd
                 alignas(avx2::alignment()) uint8_t buffer[64];
                 // FIXME: ultra inefficient
                 for (int i = 0; i < 64; ++i)
-                    buffer[i] = cond.data & ((uint64_t)1 << i) ? 0xFF : 0;
```

Comment:
This was fine

Reply:
Yeah, that was just for consistency with the remaining part of the code.